packages/appChrome/components/AppChrome.tsx

```diff
@@ -87,6 +92,7 @@ const AppChrome = ({
             {children}
           </main>
         </div>
+        {footerBar && <footer data-cy="footerBar">{footerBar}</footer>}
```

Comment:
maybe we shouldn't introduce a footerBar here if we're arguing above that headerbar is deprecated?

Reply:
headerBar is not deprecated, that is mainContent as children was introduced

Reply:
that makes way more sense :)